paymentsheet/src/main/java/com/stripe/android/paymentsheet/state/PaymentSheetLoader.kt

```diff
@@ -301,3 +286,24 @@ internal class DefaultPaymentSheetLoader @Inject constructor(
         )
     }
 }
+
+private fun List<PaymentMethod>.sorted(
```

Comment:
I think it would be beneficial to have a test for this. I would love to see a test where we initialize a list of payment methods and expect a certain sorted list of payment methods.